lib/migration.js

```diff
@@ -412,92 +397,162 @@ function mixinMigration(MySQL, mysql) {
         }
       }
     });
+    if (sql.length > 0) {
```

Comment:
The old function has a `checkOnly`, if missing `done` then just check but don't execute the sql.
Now the var is defined in L508 but not checked in this function.